spec/rapis/_files/paths.rb

```diff
@@ -0,0 +1,253 @@
+item "/" do
+  get do
+    consumes ["application/json"]
+    produces ["text/html"]
+    responses do
+      code "200" do
+        description "200 response"
+        headers(
+          {"Content-Type"=>{"type"=>"string"}})
+      end
+    end
+    amazon_apigateway_integration do
+      responses do
+        default do
+          statusCode 200
+          responseParameters(
+            {"method.response.header.Content-Type"=>"'text/html'"})
+          responseTemplates(
+            {"text/html"=>"<html></html>"})
+        end
+      end
+      requestTemplates(
+        {"application/json"=>"{\"statusCode\": 200}"})
+      passthroughBehavior "when_no_match"
+      type "mock"
+    end
+  end
+  post do
+    produces ["application/json"]
+    responses do
+      code "200" do
+        description "200 response"
+        schema do
+          ref "#/definitions/Empty"
+        end
+        headers(
+          {"Access-Control-Allow-Origin"=>{"type"=>"string"}})
+      end
+    end
+    amazon_apigateway_integration do
+      responses do
+        default do
+          statusCode 200
+          responseParameters(
+            {"method.response.header.Access-Control-Allow-Origin"=>"'*'"})
+        end
+      end
+      uri "http://petstore-demo-endpoint.execute-api.com/petstore/pets"
+      passthroughBehavior "when_no_match"
+      httpMethod "POST"
+      type "http"
+    end
+  end
+  options do
+    consumes ["application/json"]
+    produces ["application/json"]
+    responses do
+      code "200" do
+        description "200 response"
+        schema do
+          ref "#/definitions/Empty"
+        end
+        headers(
+          {"Access-Control-Allow-Origin"=>{"type"=>"string"},
```

Comment:
__[rubocop]__
* Redundant curly braces around a hash parameter.
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)
* Surrounding space missing for operator `=>`. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)